components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -107,7 +138,7 @@ export default function RepositoryFinder(props: RepositoryFinderProps) {
                 <div className="truncate w-80">{displayContextUrl(props.initialValue) || "Select a repository"}</div>
```

Comment:
I wonder if it would make sense to change this to something like "Select/Choose source/context" when the thing you're selecting isn't just a repo anymore?